vsintegration/src/FSharp.Editor/CodeLens/FSharpCodeLensService.fs

```diff
@@ -186,22 +184,30 @@ type internal FSharpCodeLensService
                                 logWarningf "Couldn't acquire CodeLens data for function %A" func
                                 return None
                         else return None
+#if DEBUG
                     with e -> 
                         logErrorf "Error in lazy line lens computation. %A" e
+#else
+                    with _ ->
+#endif
                         return None
                 }
             
             let inline setNewResultsAndWarnIfOverriden fullDeclarationText value = 
                 if newResults.ContainsKey fullDeclarationText then
+#if DEBUG
                     logWarningf "New results already contains: %A" fullDeclarationText
+#else
+                    ()
+#endif
                 newResults.[fullDeclarationText] <- value
-
             for symbolUse in symbolUses do
                 if symbolUse.IsFromDefinition then
                     match symbolUse.Symbol with
                     | :? FSharpMemberOrFunctionOrValue as func when func.IsModuleValueOrMember || func.IsProperty ->
-                        let funcID = func.FullName
-                        let fullDeclarationText = funcID
+                        let funcID = func.LogicalName + (func.FullType.ToString() |> hash |> string)
+                        // Use a combination of the the function name + the hashed value of the type signature
+                        let fullDeclarationText = funcID // (textSnapshot.GetText declarationSpan).Replace(func.CompiledName, funcID)
```

Comment:
Looks like some commented out code here. Is it important to leave, or can it just be deleted?

Reply:
I'll delete it.

Reply:
OK deleted the whole line to clear up variable names too.